pkg/util/computeskus/computeskus.go

```diff
@@ -32,3 +34,33 @@ func HasCapability(sku *mgmtcompute.ResourceSku, capabilityName string) bool {
 
 	return false
 }
+
+// FilterVmSizes filters resource SKU by location and returns only virtual machines, their names, restrictions, location info, and capabilities.
+func FilterVmSizes(skus []mgmtcompute.ResourceSku, location string) map[string]*mgmtcompute.ResourceSku {
```

Comment:
Acronyms in the middle of the name should be all caps. For example: `VMSize` in worker/master profiles.

```suggestion
func FilterVMSizes(skus []mgmtcompute.ResourceSku, location string) map[string]*mgmtcompute.ResourceSku {
```